### DISCUSSION TITLE
Xamarin has tvos support, but no "tvos" workload in Maui????

### DISCUSSION THREAD

**Discussion by pwfulghum** [dotnet/maui#6752] [General]
Early MAUI documents said tvos (Apple TV) would be supported.   But as we approach RCx, I see that only iOS is supported and not tvOS.

Xamarin has support for tvos, why not MAUI?  <sigh>

Is tvos support on the horizon?

**Comment by JohnHDev** [comment]
Xamarin Forms doesn’t support tvOS, what MS are providing is .net 6 support for tvOS.
